oauth2_http/java/com/google/auth/oauth2/ServiceAccountCredentials.java

```diff
@@ -562,6 +592,10 @@ public final URI getTokenServerUri() {
     return tokenServerUri;
   }
 
+  int getLifetime() {
```

Comment:
Can we add the `@VisibleForTesting` annotation to note that it's default visibility only for testing? (not used at runtime by another class in this package)

Reply:
done